### PR TITLE
Add quote handling and MarkdownUtil

### PR DESCRIPTION
**Pull request by MinnDevelopment**
[contributing]: https://github.com/DV8FromTheWorld/JDA/wiki/5%29-Contributing

## Pull Request Etiquette

- [x] I have checked the PRs for upcoming features/bug fixes.
- [x] I have read the [contributing guidelines][contributing].

### Changes

- [x] Internal code
- [ ] Library interface (affecting end-user code) 
- [ ] Documentation
- [ ] Other: \_____ 

Closes Issue: NaN

## Description

I've added a utility class to apply formatting and added handling for quote markdown in the sanitizer.

### CODE REVIEW

**Comment by Andre601** [comment]
Is there already known if quote blocks are still normal text in contentRaw or are they in fact embeds (considering their style is close/simlar to embeds)?

**Comment by MinnDevelopment** [comment]
image

The only similar thing with embeds is the colored column to the left of it. The background is missing though and the column is a bit different too.

**Comment by MinnDevelopment** [comment]
The handling for the sanitizer is a bit broken currently because it ignores line-breaks which are relevant for quotes. Additionally jake said they were going to change in a way that it requires a space such as `> hello` rather than `>hello`.

**Comment by Andre601** [comment]
> The handling for the sanitizer is a bit broken currently because it ignores line-breaks which are relevant for quotes. Additionally jake said they were going to change in a way that it requires a space such as `> hello` rather than `>hello`.

The latest info from Discord-Datamining does support this change.
https://github.com/DJScias/Discord-Datamining/commit/56ff440c09745aec01df087959b62059e1b6cd4f#commitcomment-34437277

File: src/main/java/net/dv8tion/jda/api/utils/MarkdownUtil.java [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+    public static String strike(@Nonnull String input)
+    {
+        String sanitized = MarkdownSanitizer.escape(input, ~MarkdownSanitizer.STRIKE);
+        return "~~" + sanitized + "~~";
+    }
+
+    /**
+     * Escapes already existing quote regions in the input
+     * and applies quote formatting to the entire string.
+     * <br>The resulting string will be {@code "> " + escaped(input).replace(LF, "> ")}.
```
Comment (MinnDevelopment) [inline src/main/java/net/dv8tion/jda/api/utils/MarkdownUtil.java]:
```suggestion
     * <br>The resulting string will be {@code "> " + escaped(input).replace("\n", "\n> ")}.
```

**Pull request merged**